Helper/Data.php

```diff
@@ -156,67 +199,77 @@ public function canAllowMultiPleCurrency()
     }
 
     /**
-     * Get currencly allowed for specific countries config
+     * Get currently allowed for specific countries config  REACH_SPECIFIC_PATH - Function 07
      *
      * @return boolean
      */
     public function allowCurrencySpeicifcCountry()
```

Comment:
can you correct this typo too? allowCurrencySpecificCountry

Reply:
No problem. There are spelling errors all over the place. Thankfully, most of them are in comments but sometimes (like this one) in function and method names.